ztp/source-crs/HardwareEvent.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: event.redhat-cne.org/v1alpha1
```

Comment:
Did you miss namespace ? This should be created under openshift-hw-events.
Also Are you sure you are allowed to create openshift-* namespace. It is a reserved namespace as far remember.

Reply:
Added namespace. Asked at wg-platform-lifecycle-5g channel and it seems to ok to use openshift-* namespace.